### ISSUE TITLE
Upload files

### ISSUE THREAD

**Issue by stevkim** [buffalo-while/Project-Atelier#94]
https://github.com/buffalo-while/Project-Atelier/blob/2493af5a61d34a3e16e3d48dd7abe657ea3e98f3/src/components/rating-reviews/components/WriteReviewPhotos.jsx#L14

Are we uploading files even if we don't post to our backend?

**Comment by stephencapper** [comment]
Yes, I heard you mention it on the call and realized I will need to refactor this.
